stravalib/tests/unit/test_model.py

```diff
@@ -88,6 +81,17 @@ def test_deserialization_edge_cases(model_class, raw, expected_value):
     assert getattr(obj, list(raw.keys())[0]) == expected_value
 
 
+def test_subscription_callback_field_names():
+    sub_callback_raw = {
+        "hub.mode": "subscribe",
+        "hub.verify_token": "STRAVA",
+        "hub.challenge": "15f7d1a91c1f40f8a748fd134752feb3",
+    }
+    sub_callback = SubscriptionCallback.parse_obj(sub_callback_raw)
+    assert sub_callback.hub_mode == 'subscribe'
+    assert sub_callback.hub_verify_token == 'STRAVA'
```

Comment:
This is moved from an existing test to ensure the `.` in the field name is replaced with `_` to make a valid python identifier.